csharp-mode-tests.el

```diff
@@ -23,22 +23,6 @@
 (require 'csharp-mode)
 (require 'package)
 
-;; development only packages, not declared as a package-dependency
```

Comment:
This is a very good change, and was also mentioned by Stefan Monnier when we entered Elpa. Right now we add packages to the users system. Kudos